src/lib/forms-schema/index.ts

```diff
@@ -45,6 +46,41 @@ const SubmissionEventsSchema = Joi.object().keys({
         secret: Joi.string().required(),
       }),
     })
+    .when('type', {
+      is: 'EMAIL',
+      then: Joi.object().keys({
+        email: Joi.alternatives([
+          Joi.string().email().required(),
+          Joi.string()
+            .regex(/^{ELEMENT:\S+}$/)
+            .required(),
+        ]),
+        emailSubjectLine: Joi.string().allow(null, ''),
+        emailTemplate: Joi.object().keys({
```

Comment:
IMO we should extract these 3 properties 
```
email
emailSubjectLine
emailTemplate
``` to a variable so they can be re-used in the PDF and EMAIL submission events